src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -89,7 +89,7 @@
 @OpenAPIDefinition(
 	info = @Info(
 		title = "OpenWiFi 2.0 RRM OpenAPI",
-		version = "1.0.0",  // TODO
+		version = "2.7.0",  // NOTE: needs manual update!
```

Comment:
Minor: Don't know much about the convention, but maybe `XXX` is even better than `NOTE`?

Reply:
Going to leave this since it seems nobody on the internet can agree what `XXX` is supposed to mean - https://stackoverflow.com/q/1452934